pkgs/by-name/sp/spl/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  pkgs,
+}:
+let
+  version = "0.3.1";
+  src = pkgs.fetchgit {
+    url = "https://git.tudbut.de/tudbut/spl";
+    rev = "v${version}";
+    hash = "sha256-Bj3voUwyqUWg1f86l/Vob+kb5jJUZmf0ZG9IxyYjgh0=";
+  };
+in
+pkgs.rustPlatform.buildRustPackage {
```

Comment:
```suggestion
rustPlatform.buildRustPackage rec {
```

Reply:
Interesting! I did originally use rec, but a friend had told me it was slow and not recommended for nixpkgs. I assume they were wrong then?

Anyways, I'll fix it shortly.

Reply:
I'm unaware of how significant the performance impact is but there are significantly more Rust packages using `rec` than `let ... in`.

Generally I think following the Rust documentation helps make the code base more consistent for everyone, and a treewide change would be ideal if there was a consensus on changing this (as well as a documentation update).